Resources/Prototypes/Hydroponics/seeds.yml

```diff
@@ -1501,3 +1501,86 @@
       Max: 10
       PotencyDivisor: 20
 
+- type: seed
+  id: azalea
+  name: seeds-azalea-name
+  noun: seeds-noun-seeds
+  displayName: seeds-azalea-display-name
+  plantRsi: Objects/Specific/Hydroponics/Flowers/azalea.rsi
+  packetPrototype: AzaleaSeeds
+  productPrototypes:
+    - FlowerAzalea
+  mutationPrototypes:
+    - azaleaBlue
+    - azaleaRed
+    - azaleaWhite
+  lifespan: 25
+  maturation: 10
+  production: 3
+  yield: 3
+  potency: 10
+  growthStages: 3
+  waterConsumption: 0.60
+
+- type: seed
+  id: azaleaBlue #parenting doesn't work here, I tried.
```

Comment:
remove comment, seeds don't have sex and so can't be parents

Reply:
:blunt: